tests/integration/start_codis.sh

```
@@ -29,7 +29,10 @@ sed -i '' -e 's|databases : 1|databases : 2|' -e 's|port : 9221|port : 8001|' -e
 #ensure both master and slave are ready
 sleep 10
 
-cd ../codis
+
+cd ../
+chmod +x ./codis
+cd codis
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)